File: Dot Net Code/AgroRent/DTOs/UserSignIn.cs
using System.ComponentModel.DataAnnotations;

namespace AgroRent.DTOs
{
    public class UserSignIn
    {
        [Required]
        [EmailAddress]
        public string Email { get; set; } = string.Empty;

        [Required]
        public string Password { get; set; } = string.Empty;
    }
}

File: Springboot Code/AgroRent/src/main/java/com/majorproj/agrorent/dto/UserSignIn.java
package com.majorproj.agrorent.dto;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class UserSignIn {
	@NotBlank(message = "Email required")
	@Email(message = "Inavlid Email")
	private String email;
	@NotBlank(message = "Password required")
	private String password; 
}
